diff --git a/src/robomongo/core/domain/MongoCollectionInfo.h b/src/robomongo/core/domain/MongoCollectionInfo.h
index 1514b6b..21eafc5 100644
--- a/src/robomongo/core/domain/MongoCollectionInfo.h
+++ b/src/robomongo/core/domain/MongoCollectionInfo.h
@@ -1,5 +1,7 @@
 #pragma once
 #include <mongo/bson/bsonobj.h>
+#include <mongo/client/dbclient.h>
+#include <mongo/bson/bsonobjbuilder.h>
 #include "robomongo/core/domain/MongoNamespace.h"
 
 namespace Robomongo
